### PR TITLE
BaSP-MR-166: member authentication

### PR DESCRIPTION
**Pull request by LucianoStradiot**
I did authentication for member entity

### CODE REVIEW

File: src/Views/login/index.js [hunk 1]
```diff
@@ -48,6 +48,7 @@ const Login = () => {
   const logUser = async (userValue) => {
     try {
       const dataResponse = await dispatch(login(userValue));
+      console.log(dataResponse);
```
Comment (Eugeniagassman) [inline src/Views/login/index.js]: you should delete this line

**Review by Eugeniagassman — APPROVED** [review]
very good job!

**Comment by Katairala** [comment]
TESTING RESULTS

Hi Luciano! 

You did a great job and  the login with the member works perfectly. However we found a bug on the validations when trying to sign up a member.

Please check the following bug report. For more detailes go to the ticket MR-197 on Trello: [Link](https://trello.com/c/oPpSJlQ3/197-bug-sign-up-validation)

Bug found:

When registering, we are allowed to enter a very current or even future date. We recommend establishing a minimum age requirement to register at the gym.

Thank you for your work! You are doing greate.

Screenshot 2023-07-04 213530

**Comment by LucianoStradiot** [comment]
[[FEATURE] - Auth Members](https://trello.com/c/Nn6UZbz1/166-feature-auth-members)

**Pull request merged**